CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# v2.1.0
+
+Reaction v2.1.0 adds minor features and performance enhancements, fixes bugs and contains no breaking changes on top of v2.0.0.
+
+This release is being coordinated with `reaction-platform` and is designed to work with the same versions of `example-storefront` and `reaction-hydra`.
+
+## Notable changes
+**Introduction of Catalyst**
+Our new [Catalyst](https://github.com/reactioncommerce/catalyst) design system has been introduced into the Reaction codebase, and is used on various components including `ConfirmDialog` and `Button`.
+
+**Tag slugs are now editable independently of tag name**
+In previous versions of Reaction, tag slugs were always saved as a slugified version of the tag name, would always change if the tag name was updated, and couldn't be changed independently of the tag name.
```

Comment:
I'd prefer to phrase this with a focus on the new functionality rather than what the old functionality was - 
```
When creating a new tag, the `slug` is initially created from the tag name. Subsequent edits to a Tag name no longer automatically change the slug and it's now possible to change the slug and the tag name independently. 
```